Formula/infer.rb

```diff
@@ -3,24 +3,25 @@ class Infer < Formula
   homepage "https://fbinfer.com/"
   # pull from git tag to get submodules
   url "https://github.com/facebook/infer.git",
-      :tag      => "v0.15.0",
-      :revision => "8bda23fadcc51c6ed38a4c3a75be25a266e8f7b4"
-
-  bottle do
-    cellar :any
-    sha256 "0b056e3162e0e5c791173f790e5e06dda2f80781531098ca8c6eb3d89dc96768" => :high_sierra
-    sha256 "91c68a2e6487e2218567a2e92c10b76bbfea5c69497b1bd9b027426ed23ec615" => :sierra
-    sha256 "8bb9d822db58e8b34e286dbc167c391e497ae5e37d96766ca355dd9bc7e6ec50" => :el_capitan
-  end
+      :tag      => "v0.16.0",
+      :revision => "4a91616390c058382c703f47653adfaecd31a7d7"
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "cmake" => :build
+  depends_on "gnu-sed" => :build
   depends_on :java => ["1.7+", :build]
   depends_on "libtool" => :build
-  depends_on "ocaml" => :build
-  depends_on "opam" => :build
+  depends_on "perl" => :build
   depends_on "pkg-config" => :build
+  depends_on "python@2" => :build
+  depends_on :x11 => :build
```

Comment:
Any idea why this is needed?

Reply:
I noticed the configure script checking for python2 actually. wasn't sure if it helps building some module of infer - do you think it can be removed?  
x11 I figured we would keep as just the build dependency since I still can't figure out editing a the pinned ocaml compiler :/

Reply:
My question was about x11 (I guess GitHub doesn't make that obvious at all...), I see now.

Python2 is indeed needed by infer, and not only at build time so `=> :build` should probably be removed for `python@2`.